multi_gpu/four/model_parallel.py

```diff
@@ -0,0 +1,162 @@
+####################
+"""Model Adapted from https://github.com/jamespengcheng/PyTorch-CNN-on-CIFAR10/blob/master/ConvNetClassifier.py"""
```

Comment:
Add an attribution statement to the main readme.  Something similar to https://github.com/CHTC/templates-GPUs/tree/master/docker/tensorflow_python#attribution